pystarport/pystarport/cosmoscli.py

```diff
@@ -159,6 +162,7 @@ def gentx(self, name, coins, min_self_delegation=1):
             home=self.data_dir,
             chain_id=self.chain_id,
             keyring_backend="test",
+            **{k: v for k, v in options.items() if v is not None},
```

Comment:
```suggestion
            pubkey=pubkey,
```

It already filter out `None` values.